README.md

```diff
@@ -32,7 +32,9 @@ and assumes the environment variables that the activation script sets):
 
     source ./env/bin/activate
 
-After having created the python virtual environment and having activated it the package
+To build the project you will also need node.js. Please read the instructions for your system [here](https://nodejs.org/en/download/package-manager/).
```

Comment:
ok I must be misremembering - I thought I might have done something like `brew install npm` but when I do `brew list` it tells me I have node but not npm. Sorry about that.